src/SphericalCoordinates_TEST.cc

```diff
@@ -114,6 +129,54 @@ TEST(SphericalCoordinatesTest, SetFunctions)
     EXPECT_EQ(sc.HeadingOffset(), heading);
     EXPECT_NEAR(sc.ElevationReference(), elev, 1e-6);
   }
+
+  // Moon surface type
+  st = math::SphericalCoordinates::MOON_SCS;
+  math::SphericalCoordinates moonSC(st);
+  moonSC.SetSurface(st);
+  EXPECT_EQ(moonSC.Surface(), st);
```

Comment:
We should test the other new get functions (e.g., `SurfaceRadius()`, `SurfaceAxisEquatorial()`, etc). First for the default parameters above (i.e., matches Earth values) then for the Moon surface type here.